registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/osivalidator/OSIValidatorStage.java

```diff
@@ -76,23 +89,16 @@
 	@Autowired
 	MosipRouter router;
 
-	@Autowired
```

Comment:
Do we need to remove this ?

Reply:
These beans are unused in OSIValidatorStage